packages/react-portal/src/components/Portal/usePortalMountNode.ts

```diff
@@ -17,7 +17,7 @@ export const usePortalMountNode = (options: UsePortalMountNodeOptions) => {
   const { targetDocument, dir } = useFluent();
 
   const element = React.useMemo(() => {
-    if (isSSR() || targetDocument === undefined || options.disabled) {
+    if (!canUseDOM() || targetDocument === undefined || options.disabled) {
```

Comment:
why can't you use `useIsSSR` here ? there's no global scope to worry about here

Reply:
Technically, we don't this check there at all as `targetDocument` will be `undefined` in SSR.

---

> why can't you use `useIsSSR` here ? there's no global scope to worry about here

- `useIsSSR()` is a hook, so I can't use inside `.useMemo()` 😯
- `useIsSSR()` can cause a re-render 😐
- we don't care there about SSR, we can if we can use DOM 😋

Reply:
> targetDocument will be undefined in SSR.

That's a good point, let's remove it... the `targetDocument` check is for TS to narrow the type from `undefined`. I don't really see a case for `canUseDOM` at least in components since you'd be pulling the target document from fluent context

Reply:
Done 😉